Source/WebCore/Modules/modern-media-controls/controls/layout-traits.js

```diff
@@ -36,6 +36,11 @@ class LayoutTraits
         return this.mode == LayoutTraits.Mode.Fullscreen;
     }
 
+    isApplePlatform()
+    {
+        return true;
```

Comment:
i don't think we want to assume that all subclasses are Apple (or support control bar overflow), so can we have it `throw` and then add this to all the existing subclasses?